lib/kafo/puppet_log_parser.rb

```diff
@@ -1,9 +1,5 @@
 module Kafo
   class PuppetLogParser
-    def initialize
-      @last_level = nil
-    end
```

Comment:
I'm hesitant to do this since it can break multi line logging (which was introduced in 053b7c02a05679da45d5307bafe464a2a17bda91). Is this really needed?

Reply:
Is there some example of multi line logging which is not a traceback ?